### DISCUSSION TITLE
async for /home?

### DISCUSSION THREAD

**Discussion by kvaps** [ltsp/ltsp#210] [Q&A]
Hi, just found this, but I can’t recognize is it a bug or feature:

https://github.com/ltsp/ltsp/blob/046af079fc85a0c6e0bd90c2bf1222265f0f9505/ltsp/server/nfs/ltsp-nfs.exports#L17

Is it safe to export /home with `async` option?

**Comment by alkisg** [comment]
It improves performance but might cause issues on **unclean** server restart while the clients are running.

In LTSP, we expect the server to always be up (not considering IMAGE_TO_RAM). E.g. with the default SSHFS, if the server restarts even cleanly, the clients won't reconnect /home anyway, so they'll even need to be logged out.

So having better performance sounds better than trying to cover the corner case where the server crashes and the administrator expects the clients to continue working even without logout. And they might be able to do even that; but yeah, with possible loss of the last written data.

I've transferred this to the community issues to leave it open in case LTSP administrators would like to mention that they prefer otherwise.
